mlir/include/mlir/Dialect/XeGPU/IR/XeGPUOps.td

```diff
@@ -595,6 +604,16 @@ def XeGPU_PrefetchOp : XeGPU_Op<"prefetch", []> {
     As compared to prefetch_nd, which works on non-scattered TensorDesc,
     it works on scattered TensorDesc instead.
```

Comment:
Nit: could you help to update the doc here. It seems the definition has switched to use memref/pointer instead of TensorDesc

Reply:
`XeGPU_GatherScatterSourceType` does contain `XeGPU_TensorDesc`
Check here:
https://github.com/silee2/llvm-project/blob/updateScatterDesc/mlir/include/mlir/Dialect/XeGPU/IR/XeGPUTypes.td#L196-#L197